lib/src/services/score_service.dart

```diff
@@ -0,0 +1,38 @@
+import 'dart:convert';
+
+import 'package:flutter/services.dart';
+import '../models/score/score_model.dart';
+
+class ScoreService {
+  ScoreService._();
+
+  static final ScoreService _instance = ScoreService._();
+
+  static ScoreService get instance => _instance;
+
+  Future<List<ScoreModel>> forUser(String userId) async {
+    return (await all()).where((score) => score.userUid == userId).toList();
+  }
+
+  Future<Map<String, int>> mappedToUid() async {
+    Map<String, int> uidScoreMap = {};
+
+    for (var score in (await all())) {
```

Comment:
kann man so machen. 

Ich würde aber keine zwei Anweisungen in eine Zeile packen.
Lieber die extra Zeile var scores = await (find)all();

dann liest es sich auch glatter 

for(var score in scores){....